src/syft/core/store/storeable_object.py

```diff
@@ -147,27 +165,13 @@ def _proto2object(proto: StorableObject_PB) -> Serializable:
         if not isinstance(id, UID):
             traceback_and_raise(ValueError("TODO"))
 
-        # TODO: FIX THIS SECURITY BUG!!! WE CANNOT USE
-        #  PYDOC.LOCATE!!!
-        # Step 2: get the type of wrapper to use to deserialize
-        obj_type: StorableObject = pydoc.locate(proto.obj_type)  # type: ignore
-
-        # this happens if we have a special ProtobufWrapper type
-        # need a different way to get obj_type
-        if proto.obj_type.endswith("ProtobufWrapper"):
-            module_parts = proto.obj_type.split(".")
-            klass = module_parts.pop().replace("ProtobufWrapper", "")
-            proto_type = getattr(sys.modules[".".join(module_parts)], klass)
-            obj_type = proto_type.serializable_wrapper_type
+        # # TODO: FIX THIS SECURITY BUG!!! WE CANNOT USE
+        # #  PYDOC.LOCATE!!!
+        # # Step 2: get the type of wrapper to use to deserialize
+        data_type = pydoc.locate(proto.data_type)
```

Comment:
Ideally here we should be able to use the AST to solve the types, the main problem being that we don't have the wrapper types in our AST. Do you think we can find a workaround on pydoc.locate (maybe add the wrapper types in the AST/in some kind of cache). Not a must for this PR, but it's something worth to think on.

Reply:
I think we can use function `index_syft_by_module_name`. I already add wrapper classes to AST path. But not really add them, just fake it like it's there. Please take a look inside `index_syft_by_module_name`. For exmaple, torch.deviceWrapper has a ast path syft.wrappers.torch.deviceWrapper